### PR TITLE
Tests KIVA fixes

### PR DESCRIPTION
**Pull request by brgix**
Testing fixes introduced in this TBD [PR](https://github.com/rd2/tbd/pull/120).

### CODE REVIEW

File: tbd.schema.json [hunk 1]
```diff
@@ -267,15 +267,15 @@
           "type": "number"
         },
         "corner": {
-          "title": "Any wall corner [0°,135°) or (225°,360°] PSI",
+          "title": "Any wall corner [0deg,135deg) or (225deg,360deg] PSI",
```
Comment (brgix) [inline tbd.schema.json:270]: Linked to this [comment](https://github.com/rd2/tbd/pull/120#discussion_r1850927409). Fix works for both SDK v3.8.0 & v3.9.0.

More fixing to do ...

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -157,5 +157,4 @@ jobs:
         docker exec -t test ls
         docker exec -t test bundle update
         docker exec -t test bundle exec rake
-        docker exec -t test bundle exec rake suites_run
```
Comment (brgix) [inline .github/workflows/pull_request.yml]: Unable to isolate the _OpenStudio-Standards_ gem issue (prototype test suite) for now. OK with skipping the _prototype_ test suite altogether when it comes to a _bleeding edge_ version of the OpenStudio/SDK (v3.9.0 was released last week). Can certainly wait for the next release.

File: spec/tbd_tests_spec.rb [hunk 3]
```diff
@@ -11033,6 +11033,45 @@
     expect(model).to_not be_empty
     model = model.get
 
+    # Fetch all 5 outdoor-facing walls of the Open Area space.
```
Comment (brgix) [inline spec/tbd_tests_spec.rb:11036]: Harmonized with test changes [here](https://github.com/rd2/tbd/pull/120#discussion_r1853052646).

File: tbd_tests.gemspec [hunk 4]
```diff
@@ -52,8 +52,8 @@ Gem::Specification.new do |s|
 
     s.add_development_dependency "bundler",     "~> 2.4.10"
 
-    s.add_development_dependency "openstudio-common-measures",    "~> 0.10.0"
-    s.add_development_dependency "openstudio-model-articulation", "~> 0.10.0"
+    s.add_development_dependency "openstudio-common-measures",    "~> 0.11.0"
+    s.add_development_dependency "openstudio-model-articulation", "~> 0.11.0"
```
[the remark this answers is not in the document]
Comment (macumber) [reply tbd_tests.gemspec:56]: This is a good update though

File: lib/measures/tbd_monkey_patch/measure.rb [hunk 5]
```diff
@@ -22,7 +22,7 @@
 
 require 'openstudio-standards'
 
-if OpenstudioStandards::VERSION >= '0.6.0' and OpenstudioStandards::VERSION <= '0.6.3'
+if OpenstudioStandards::VERSION >= '0.6.0'
```
Comment (brgix) [inline lib/measures/tbd_monkey_patch/measure.rb:25]: Works locally. All good. Thanks @macumber
Hadn't re-checked this, as it's working just fine for SDK v3.8.0.

**Pull request merged**